framework/src/modules/interoperability/utils.ts

```diff
@@ -88,3 +88,31 @@ export const handlePromiseErrorWithNull = async <T>(promise: Promise<T>) => {
 	}
 	return result;
 };
+
+export const isNullCharacterIncluded = (input: string): boolean =>
+	new RegExp(/\\0|\\u0000|\\x00/).test(input);
+
+export const isValidName = (username: string): boolean => {
```

Comment:
I think it also needs to check the case `if (username.length > MAX_LENGTH_NAME)` as per the LIP unless it's being already checked somewhere else.